src/test/kotlin/org/treeWare/mySql/operator/CreateDatabaseTests.kt

```diff
@@ -17,19 +16,23 @@ import kotlin.test.assertEquals
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 class CreateDatabaseTests {
-    private val dbServer: MySQLContainer<Nothing>
     private val connection: Connection
 
     init {
-        dbServer = MySQLContainer<Nothing>(DockerImageName.parse("mysql:8.0.29"))
-        dbServer.start()
-        connection = DriverManager.getConnection(dbServer.jdbcUrl, "root", dbServer.password)
+        connection = DriverManager.getConnection(
+            MySqlTestContainer.dbServer.jdbcUrl, "root",
+            MySqlTestContainer.dbServer.password
+        )
+
+        connection.autoCommit = false
     }
 
     @AfterAll
     fun afterAll() {
-        dbServer.stop()
+        clearDatabase(connection, "test\$address_book")
```

Comment:
nit: Looks like the test in this file creates a local variable for it, while other files create a constant at the top of the file (probably because those files have multiple tests in them). Create a constant for the database name at the top of the file like the other files.

Applies to other files as well that don't have a top-level constant.